WordPress/Classes/ViewRelated/Media/MediaLibraryViewController.swift

```diff
@@ -83,7 +83,7 @@ class MediaLibraryViewController: WPMediaPickerViewController {
 
         title = NSLocalizedString("Media", comment: "Title for Media Library section of the app.")
 
-        automaticallyAdjustsScrollViewInsets = false
+        collectionView?.contentInsetAdjustmentBehavior = .never
```

Comment:
Interestingly, this change makes the scroll indicator disappear. Reverting it shows the scroll indicator.

From `develop`:
<img width="431" alt="indicator" src="https://user-images.githubusercontent.com/1816888/58732037-968e4d80-83ad-11e9-890e-a66e58443a75.png">

I'm not sure it's worth holding up this change though. I'll leave it to you; just let me know how you'd like to proceed.

Reply:
Well spotted!  It looks like it might be running behind the search bar when its scrolled to the top.  Setting the property to `.automatic` seems to resolve the issue.   Since this is the default value I think we can just get rid of the line altogether.

Reply:
Updated! Ready for another peek!